### PR TITLE
App deployer: Updating logs for errors

### PR DESCRIPTION
**Pull request by oumkale**
Signed-off-by: Oum Kale <oumkale@chaosnative.com>

App deployer: Updating logs for errors

### CODE REVIEW

**Review by ishangupta-ds — APPROVED** [review]

File: custom/workflow-helper/app-deployer/main.go [hunk 1]
```diff
@@ -36,17 +37,17 @@ func main() {
 	//GetData is initializing required variables for app-deployer
 	appVars, err := GetData()
 	if err != nil {
-		panic(err.Error())
+		logs.Fatalf("err: %v", err)
```
Comment (ksatchit) [inline custom/workflow-helper/app-deployer/main.go:40]: can we have a more detailed log message here?

Comment (oumkale) [reply custom/workflow-helper/app-deployer/main.go:40]: We have been handling it in functions : 
```return &appVars, fmt.Errorf("app '%v' not supported in app-deployer", appVars.app)```

Comment (oumkale) [reply custom/workflow-helper/app-deployer/main.go:40]: return &appVars, fmt.Errorf("app '%v' not supported in app-deployer", appVars.app)

Comment (oumkale) [reply custom/workflow-helper/app-deployer/main.go:40]: [return &appVars, fmt.Errorf("app '%v' not supported in app-deployer", appVars.app)](url)

**Review by gdsoumya — APPROVED** [review]

**Pull request merged**